fitbenchmarking/fitbenchmarking
# Azure test incorrect report

#### Description of Work

Fixes #595 

Corrects reports successes and failures for the window tests in azure.

#### Testing Instructions

1. Check that the tests correctly pass on azure
2. Check (via previous commits) that azure reports failures correctly

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
